--- origsrc/xpaint-2.8.8/Imakefile	2009-11-07 04:21:33.000000000 -0600
+++ src/xpaint-2.8.8/Imakefile	2009-12-17 22:17:46.823835900 -0600
@@ -94,7 +94,7 @@ SGI_LIB = -limage
 DEPLIBS = always xpaint.man XPaint.ad
 
 #if ! defined(__NetBSD__) && !defined(__FreeBSD__)
-LOCAL_LIBRARIES = -Lrw -lrw $(XPM_LIB) $(TIFF_LIB) $(SGI_LIB) $(JPEG_LIB) $(PNG_LIB) -ldl -rdynamic
+LOCAL_LIBRARIES = -Lrw -lrw $(XPM_LIB) $(XTOOLONLYLIB) $(XMULIB) $(EXTENSIONLIB) $(XONLYLIB) $(TIFF_LIB) $(SGI_LIB) $(JPEG_LIB) $(PNG_LIB) -ldl -rdynamic
 #else
 LOCAL_LIBRARIES = -Lrw -lrw $(XPM_LIB) $(TIFF_LIB) $(SGI_LIB) $(JPEG_LIB) $(PNG_LIB)
 #endif
--- origsrc/xpaint-2.8.8/graphic.c	2009-12-08 15:16:59.000000000 -0600
+++ src/xpaint-2.8.8/graphic.c	2009-12-17 22:10:26.364643100 -0600
@@ -5564,7 +5564,7 @@ StdSnapshotCallback(Widget w, XtPointer 
             rect.height = pw->paint.drawHeight;
 	}
         zoomUpdate(pw, True, &rect);
-#if defined XAW3D && !defined NEXTAW
+#if defined XAW3DG && !defined NEXTAW
         /* redraw border of viewport window with Xaw3d widgets ... */
         XtVaGetValues((Widget)
                ((ViewportWidget)(info->viewport))->viewport.threeD, 
