--- origsrc/xpaint-2.9.9/util/pnm.cpp	2010-08-06 03:57:25.000000000 -0500
+++ src/xpaint-2.9.9/util/pnm.cpp	2012-10-31 01:40:36.587247400 -0500
@@ -159,7 +159,9 @@ bool CPNM::ReadPNM(CPGFImage& pgf, int q
 	header.mode = m_mode;
 	header.nLevels = levels;
 	header.quality = quality;
+#if PGFCodecVersionID < 0x061142
 	header.background.rgbtBlue = header.background.rgbtGreen = header.background.rgbtRed = 255;
+#endif
 	pgf.SetHeader(header, (roi) ? PGFROI : 0);
 
 	// read image data depending on image type
